docs/docs/layouts/bar.md

```diff
@@ -0,0 +1,49 @@
+A thin module which appears at the top or bottom of the browser window. Note that the bar module does not support header text.
```

Comment:
I think the note should be called out on it's own line. It should also be phrased in terms of the API: "**Note:** the bar module has no headline config property".